src/test/java/teammates/test/cases/storage/StudentsDbTest.java

```diff
@@ -33,6 +35,38 @@ public static void setupClass() throws Exception {
     }
     
     @Test
+    public void testDefaultTimestamp() throws InvalidParametersException, EntityAlreadyExistsException, EntityDoesNotExistException {        
+        ______TS("success : created");
+        
+        StudentAttributes s = new StudentAttributes();
+        s.name = "a valid student";
+        s.lastName = "last name of student";
+        s.email = "valid-fresh-student@email.com";
+        s.team = "validTeamNameOfStudent";
+        s.section = "aValidSectionName";
+        s.comments = "";
+        s.googleId = "validGoogleIdForStudent";
+        s.course = "valid-course";
+        studentsDb.createEntity(s);
+        
+        StudentAttributes student = studentsDb.getStudentForGoogleId(s.course, s.googleId);
+        assertNotNull(student);
+        
+        student.setCreated_NonProduction(null);
+        student.setUpdatedAt_NonProduction(null);
+        
+        Date defaultStudentCreationTimeStamp = Const.TIME_REPRESENTS_DEFAULT_STUDENT_CREATION_TIMESTAMP;
+        
+        ______TS("success : created defaultTimeStamp");
+        
+        assertEquals(defaultStudentCreationTimeStamp, student.getCreatedAt());
+        
+        ______TS("success : update defaultTimeStamp");
```

Comment:
The test case description is misleading here. Write as "defaultTimeStamp for updatedAt date" instead. Similarly for the createdAt date test case.